cdn.php

```diff
@@ -43,7 +43,9 @@ public function onOutputGenerated()
         }
 
         // set the protocol to HTTPS if you access that way
-        if (isset($_SERVER['HTTPS']) && strtolower($_SERVER['HTTPS']) == 'on') {
+        if ( (isset($_SERVER['HTTPS']) && strtolower($_SERVER['HTTPS']) == 'on') ||
+           ( isset($config['forcehttps']) && $config['forcehttps'] == 1 ))
```

Comment:
I think this should be:

```
 isset($config['forcehttps']) && $config['forcehttps'] == true )
```

as these settings are typically true/false and the blueprint being `validate: type: bool` is setting this to be true/false

Reply:
I'll take a look at it. 

Originally I wrote it like this:

`isset($config['forcehttps']) && $config['forcehttps'] )`

but that didn't work, thus == 1. I'll take a look at if == true works.

Reply:
== true works, pushed new version to the branch.